source/scenario/scenario3e.vwf.yaml

```diff
@@ -129,6 +129,25 @@ children:
           - writeToBlackboard:
             - Level3eTravel1
 
+        failOnMissingBlock_3e:
+          group: successOrFailure
+          priority: 3
+          triggerCondition:
+          - or:
+            - onBlockMissing:
+              - rover
+              - controls_repeat_ext
+            - onBlockMissing:
+              - rover
+              - controls_whileUntil
+            - onBlockMissing:
+              - rover
+              - controls_sensor_metal
+          actions:
+          - scenarioFailure:
+            - incomplete
+            - "You must next the loop block inside the conditional loop block."
```

Comment:
next => nest?

Although, as I said above, we should think over these messages regardless...